docs/cli.md

```diff
@@ -66,14 +66,14 @@ Available formats include:
 
 An example of using `-o template` to retrieve the *name* of the first build:
 
-`$ openshift cli get builds -o template -t "{{with index .items 0}}{{.metadata.name}}{{end}}"`
+`$ osc get builds -o template -t "{{with index .items 0}}{{.metadata.name}}{{end}}"`
 
 ### Selectors
 
 cli `get` provides also 'selectors' that you can use to filter the output
```

Comment:
Shouldn't all occurrences of 'cli' be changed to 'osc'? There's a good number of them i n this file